### DISCUSSION TITLE
Welcome to gopherdaily Discussions!

### DISCUSSION THREAD

**Discussion by bigwhite** [bigwhite/gopherdaily#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hakc12** [comment]
It will a lot of help for many people add a feature to upload that results folder to custom address and accessible by attacker such that address can be changed or modified by attacker I hope you will consider my feature and do more of discussion and your opinion and thoughts about it
